llvm/tools/llvm-cas/llvm-cas.cpp

```diff
@@ -50,20 +59,22 @@ static int diffGraphs(ObjectStore &CAS, const CASID &LHS, const CASID &RHS);
 static int traverseGraph(ObjectStore &CAS, const CASID &ID);
 static int ingestFileSystem(ObjectStore &CAS, std::optional<StringRef> CASPath,
```

Comment:
Please merge your `ingestFile` with `ingestFileSystem`. We only need one that takes a list of files (or directories) and ingest them all into CAS.